### ISSUE TITLE
Graph page not working

### ISSUE THREAD

**Issue by NOV-DM** [socialpandas/sidekiq_monitor#9]
I get the following error in my browser console when i go to graph page:

![image](https://f.cloud.github.com/assets/3205115/1129765/b96ee176-1b93-11e3-94a7-0bae7955924b.png)

**Comment by tombenner** [comment]
A `/#!` shouldn't be included in the requested URLs. You likely have another plugin that's interfering here. Please try entering the following into your JS console:

```
SidekiqMonitor.settings.api_url('')
```

If this returns `/sidekiq/api`, then the issue is probably with a redirect; if it returns `/#!/sidekiq/api`, then the issue is related to something rewriting the route for where Sidekiq Monitor is mounted.

**Comment by NOV-DM** [comment]
`/#!` is included because there was no matching route found in rails. I am using rails for json api and angularjs for frontend. If the api route is not found it tries to redirect it to the frontend.

```
SidekiqMonitor.settings.api_url('')
```

returns 

```
/sidekiq/api
```

I looked into it a bit and the problem is that I mount my sidekiq_monitor to 

```
mount Sidekiq::Monitor::Engine => '/sidekiq/monitor'
```

but 

```
SidekiqMonitor.settings.api_url('')
```

still returns 

```
 /sidekiq/api
```

it should return 

```
 /sidekiq/monitor/api
```

i tried to call 

```
http://localhost:9000/sidekiq/monitor/api/jobs/graph
```

and the response was correct

I tried a temporary workaround and mounted my sidekiq_monitor to 

```
mount Sidekiq::Monitor::Engine => '/sidekiq'
```

and it works but the problem is that I also have sidekiq mounted to the same route so now when I access that url i get to sidekiq and if i want to access sidekiq_monitor tabs I have to manually go to url like 

```
/sidekiq/graph
```

but other pages like /queues and /jobs is not working because it is using sidekiq tabs instead
